### DISCUSSION TITLE
Trying to get started:  "Error reaching Varnish"

### DISCUSSION THREAD

**Discussion by lilbumblebear** [scandipwa/scandipwa#2545] [Install & Upgrade]
I have the scandipwa theme installed as mentioned in the instructions for an existing M2 instance.  I updated the proxy to use our local instance, but when I try to run setup:upgrade I get this error:

Error reaching Varnish: Unable to connect to host.local:80 . Error #0: stream_socket_client(): unable to connect to host.local:80 (php_network_getaddresses: getaddrinfo failed: Name or service not known)

Does anyone know what is causing the error?  Is there some configuration that I missed?  Just trying to get setup with yarn start and developing the theme. 

Thank you!

**Comment by alfredsgenkins** [comment]
Theme is unrelated to varnish. You are having some trouble on Magento 2 side. Is your varnish configured? Is Magento 2 started alongside the varnish? Try CMA if setting up on your own server lead to some errors.

**Comment by Kos-Adaptive** [comment]
> I have the scandipwa theme installed as mentioned in the instructions for an existing M2 instance. I updated the proxy to use our local instance, but when I try to run setup:upgrade I get this error:
> 
> Error reaching Varnish: Unable to connect to host.local:80 . Error #0: stream_socket_client(): unable to connect to host.local:80 (php_network_getaddresses: getaddrinfo failed: Name or service not known)
> 
> Does anyone know what is causing the error? Is there some configuration that I missed? Just trying to get setup with yarn start and developing the theme.
> 
> Thank you!

Hello,

Did you solve the issue? I have the same problem and can't figure out how to manage it (

Kind regards,
Konstantin

**Reply by Kos-Adaptive** [reply]
Solution:

add next strings to the env.php to the end of file:

 'http_cache_hosts' => [
        [
            'host' => '127.0.0.1'
        ]
    ]
